wremnants/datasets/datagroups.py

```diff
@@ -437,18 +440,44 @@ def setSelectOp(self, op, processes=None):
                 raise ValueError(f"In setSelectOp(): process {proc} not found")
             self.groups[proc].selectOp = op
 
-    def setGenAxes(self, gen_axes):
+    def setGenAxes(self, gen_axes=None):
         if isinstance(gen_axes, str):
-            self.gen_axes = [gen_axes,]
-        else:
+            gen_axes = [gen_axes]
+
+        if gen_axes != None:
             self.gen_axes = gen_axes
+        else:
```

Comment:
It seems to me that this would be better outside of the datagroups class

Reply:
But where would you put it?